### ISSUE TITLE
Trip being part of Trip + isPartOf extended beyond CreativeWork via Pending proposal

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1968]
@RichardWallis I'm not super happy with this change to the core, which snuck through as part of a *pending* proposal for touristy stuff. It is entirely reasonable to want to relate trips in this way, but we should not have made a core change to the (currently 100% CreativeWork-centric) isPartOf property as part of the tourism proposal. My bad for not catching this earlier.

```    <div typeof="rdf:Property" resource="http://schema.org/hasPart">
      <span class="h" property="rdfs:label">hasPart</span>
      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/Trip">Trip</a></span>
      <span>rangeIncludes: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Trip">Trip</a></span>
      <span>Category: <span property="schema:category">issue-1810</span></span>
    </div>
    <div typeof="rdf:Property" resource="http://schema.org/isPartOf">
      <span class="h" property="rdfs:label">isPartOf</span>
      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/Trip">Trip</a></span>
      <span>rangeIncludes: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Trip">Trip</a></span>
      <span>Category: <span property="schema:category">issue-1810</span></span>
    </div>
```

This looks like a bug to me. Many times we've had this discussion and there has never been consensus in the community or steering group to generalize isPartOf to other entity types. At this point I always point at https://plato.stanford.edu/entries/mereology/ and say "do we really want to go there"? I continue to believe we don't want to go there.

@rvguha can you give some KR oversight here please. The desire is for our existing Trip type to have a pattern for trips being parts of trips.

I believe we have prior consensus 

From https://github.com/schemaorg/schemaorg/issues/1097#issuecomment-209233886 in 2016,

> Richard: At first glance the basic proposal of having Thing in the range of isPartOf
that raises a +1 from me.
> Vicki: Do we have a use case for this change? I hesitate to make such a large change without a compelling use case.
> [Dan](https://github.com/schemaorg/schemaorg/issues/1097#issuecomment-209014851): 
My proposal would be to define dedicated properties for specific use cases e.g. for food we might have containsChemical, for places we already have containsPlace, for people who are "part of" organizations we have already got "affiliation", and for sub-events within a larger/longer containing event, we already have subEvent. 
> Guha: I tend to agree with Vicki and Dan. I would really like to avoid going down the path of making Thing the domain and range of too many properties.

The question to me seems to be - do we handle trips via Event, or do they have a good reason to be handled separately as an Intangible. There have been reasonable questions asked on the mailing list about this recently too.

**Comment by RichardWallis** [comment]
@danbri I see where you are coming from....

We could go with _subTrip_, but I hesitate to suggest _superTrip_ (it sounds like an endorsement to take the trip!) perhaps _subTrip_ and _partOfTrip_ would be a better pair.

As to handling _Trip_ via _Event_ instead of _Intangible_ , I wasn't entirely convinced by the reference you mentioned which seemed to suggest that many, many things were really Event subtypes.

A trip, in the travel world can often be described as an itinerary of destinations only potentially becoming an actual event, if tickets are purchased.  Purchase an around the world ticket; the bus trip from one city to another, can be described, with provider etc. that happens everyday; the [recommended] TouristTrip you should take across Australia if you had the inclination and finance; etc., etc.

**Comment by thadguidry** [comment]
@danbri The "agreement" we sorta had with the community was that we would add support for "legs of a Trip".  How that surfaces into Types or Properties is what needs to be sorted in this issue.  I am open to anything, as long as we can support "multiple connected itineraries", or "legs of a Trip".  I'd vote for any simple relationship pairing.

**Comment by danbri** [comment]
@RichardWallis - I agree, if you're not careful, you end up seeing the entire universe of human experience through the "event" lens, which is a lot of pressure to put on one data structure. 

@rvguha - what do you think about subTrip for multi-hop Trip "legs", instead of overloading the CreativeWork-centric isPartOf?

**Comment by RichardWallis** [comment]
Created new branch based upon the v3.4-release tagged version
It contains:
- domainIncludes & rangeIncludes of _hasPart_ & _isPartOf_ **now** only contain _CreativeWork_
-  Add two new properties with domainIncludes & rangeIncludes of _Trip_:
   -   _subTrip_ 
   -   _partOfTrip_
- Regenerated release files

This could be used to hotfix the current release and then be merged into master.
/cc @danbri

**Comment by danbri** [comment]
@RichardWallis thanks! Note that we do not hotfix vocabulary, only software issues and bugs. So don't regenerate a release, this will be in 3.5. The subTrip/partOfTrip properties should be in Pending, are they?

**Comment by danbri** [comment]
What's the status of this? Can you make a PR against master/ branch for v3.5, including a note for docs/releases.html

**Comment by RichardWallis** [comment]
Implemented in [release 3.5 ](https://schema.org/docs/releases.html#v3.5)
